theatre/studio/src/panels/SequenceEditorPanel/RightOverlay/FocusRangeZone/FocusRangeStrip.tsx

```diff
@@ -54,6 +54,21 @@ const RangeStrip = styled.div`
     cursor: grabbing !important;
   }
   ${pointerEventsAutoInNormalMode};
+
+  /* covers the one pixel space between the focus range strip and the top strip
```

Comment:
The glitch:

https://user-images.githubusercontent.com/593118/166226779-98d4de50-0341-40c1-b728-bf124fc0b201.mp4